doc/TROUBLESHOOT.md

```diff
@@ -101,6 +101,13 @@ Otherwise, the reference to the first non-needed object on the path (`staticX`,
 staticX -> A -> B -> disposedD
 ```
 
+Be careful not to fall into pattern to limit your fix to just releasing the reference to the disposed object. 
+Instead, you want to release most left that is not needed any more (like `staticX`, `A` or `B`).
```

Comment:
I'm not sure what you mean be "most left" here.  Do you mean that you want to release as much as possible that remains, or do you mean you want to release the leftmost (most specific) items?